qa/suites/upgrade/octopus-x/parallel/0-cluster/start.yaml

```diff
@@ -0,0 +1,43 @@
+meta:
+- desc: |
+   Run ceph on two nodes,
+   Use xfs beneath the osds.
+   CephFS tests running on client 2,3
+roles:
+#- - host.a #uncomment if mon.a mgr.x mon.b commented out
+- - mon.a #temp as roleless disabled
+  - mgr.x #temp as roleless disabled
+  - mds.a
+  - osd.0
+  - osd.1
+  - osd.2
+  - osd.3
+  - client.0
+  - client.1
+- - host.b
```

Comment:
@yuriw please comment this out too, when not using roleless, we don't need host.x